docker/1.4.1/Dockerfile.cpu

```diff
@@ -70,4 +70,4 @@ RUN chmod +x /usr/local/bin/dockerd-entrypoint.py
 EXPOSE 8080 8081
 ENV TEMP=/home/model-server/tmp
 ENTRYPOINT ["python", "/usr/local/bin/dockerd-entrypoint.py"]
-CMD ["serve"]
+CMD ["mxnet-model-server", "--start", "--mms-config", "/home/model-server/config.properties"]
```

Comment:
To summarize,

Previously in DLC(not SM) if user does `docker run <>` the docker images ran `mxnet-model-server --start --mms-config /home/model-server/config.properties` command as part of default entrypoint.  With SM integration, the default entrypoint is changed to serving.main() which essentially starts the MMS server(expecting model at /opt/ml/model) and also does some sagemaker specifc setup like setting up environment variables. 

This change is introduced to avoid the sagemaker setup while running `docker run <>` because exsiting DLC customers might not expect that